src/app/helptext/system/alert-service.ts

```diff
@@ -151,25 +151,52 @@ export default {
     SNMPTrap_port_tooltip: T(''),
 
     SNMPTrap_v3_placeholder: T('SNMPv3 Security Model'),
-    SNMPTrap_v3_tooltip: T(''),
+    SNMPTrap_v3_tooltip: T('Enable the SNMPv3 security model.'),
 
     SNMPTrap_v3_username_placeholder: T('Username'),
-    SNMPTrap_v3_username_tooltip: T(''),
-
-    SNMPTrap_v3_authkey_placeholder: T('Secret authentication key'),
-    SNMPTrap_v3_authkey_tooltip: T(''),
-
-    SNMPTrap_v3_privkey_placeholder: T('Secret encryption key'),
-    SNMPTrap_v3_privkey_tooltip: T(''),
-
-    SNMPTrap_v3_authprotocol_placeholder: T('Authentication protocol'),
-    SNMPTrap_v3_authprotocol_tooltip: T(''),
-
-    SNMPTrap_v3_privprotocol_placeholder: T('Encryption protocol'),
-    SNMPTrap_v3_privprotocol_tooltip: T(''),
+    SNMPTrap_v3_username_tooltip: T('Username of the SNMP \
+ <a href="http://snmplabs.com/pysnmp/docs/api-reference.html#user-based" \
+ target="_blank">User-based Security Model (USM)</a> user.'),
+
+    SNMPTrap_v3_authkey_placeholder: T('Secret Authentication Key'),
+    SNMPTrap_v3_authkey_tooltip: T('Initial secret authentication key. \
+ When an authentication key is not set, no <i>Authentication Protocol</i> is \
+ used. If an authentication key is set and an \
+ <i>Authentication Protocol</i> is not specified, \
+ <a href="http://snmplabs.com/pysnmp/docs/api-reference.html#pysnmp.hlapi.usmHMACMD5AuthProtocol" \
+ target="_blank">MD5</a> is used as the default.<br> \
+ Must be at least 8 and at most 32 octets long.'),
+
+    SNMPTrap_v3_privkey_placeholder: T('Secret Encryption Key'),
+    SNMPTrap_v3_privkey_tooltip: T('Initial secret encryption key. \
+ If an encryption key is not set, no <i>Encryption Protocol</i> is used. \
+ If an encryption key is set and an <i>Encryption Protocol</i> is \
+ not specified, \
+ <a href="http://snmplabs.com/pysnmp/docs/api-reference.html#pysnmp.hlapi.usmDESPrivProtocol" \
+ target="_blank">DES</a> is used as the default.<br> \
+ Must be at least 8 and at most 32 octets long.'),
+
+    SNMPTrap_v3_authprotocol_placeholder: T('Authentication Protocol'),
+    SNMPTrap_v3_authprotocol_tooltip: T('Authentication protocol \
+ used to authenticate messages sent on behalf of the specified \
+ <i>Username</i>. Here is a \
+ <a href="http://snmplabs.com/pysnmp/docs/api-reference.html#pysnmp.hlapi.usmNoAuthProtocol" \
+ target="_blank">list of supported authentication protocols</a>.'),
+
+    SNMPTrap_v3_privprotocol_placeholder: T('Encryption Protocol'),
+    SNMPTrap_v3_privprotocol_tooltip: T('Encryption protocol \
+ used to encrypt messages sent on behalf of the specified \
+ <i>Username</i>. Supported encryption protocols are shown at \
+ <a href="http://snmplabs.com/pysnmp/docs/api-reference.html#pysnmp.hlapi.usmNoPrivProtocol" \
+ target="_blank">http://snmplabs.com/</a>'),
```

Comment:
Can do the same thing here as above: make _Encryption protocol_ in the first sentence the link from the second, then remove the second.